apps/core/client/queries/get-raw-web-page-content.ts

```diff
@@ -1,25 +1,30 @@
-import { cache } from 'react';
-
 import { client } from '..';
 import { graphql } from '../generated';
 
 export const GET_RAW_WEB_PAGE_CONTENT_QUERY = /* GraphQL */ `
   query getRawWebPageContent($id: ID!) {
     node(id: $id) {
+      __typename
       ... on RawHtmlPage {
         htmlBody
       }
     }
   }
 `;
 
-export const getRawWebPageContent = cache(async (id: string) => {
+export const getRawWebPageContent = async (id: string) => {
```

Comment:
No `cache`?

Reply:
React's `cache` can only be used in RSC. Not sure how this was working on a route handler. Probably some next patching 🏛️.